src/clients/iamService.js

```
@@ -0,0 +1,149 @@
+        return false;
+      }
+      throw error;
+    }
+  }
+
+  async createIAMUser(accountId, username) {
+    try {
+      logger.info(`Creating IAM user ${username} in account ${accountId}`);
+
+      // Get IAM client for target account
+      const assumedIamClient = await this.getIAMClientForAccount(accountId);
```

ESLint

[error] 'assumedIamClient' is assigned a value but never used.

(no-unused-vars)